src/site/content/en/handbook/markup-cards-collection/index.md

```diff
@@ -19,6 +19,8 @@ YAML properties available for displaying a layout of cards:
 - `title` - the title of the card.
 - `description` - the description of the card.
 - `thumbnail` - the thumbnail of the card.
+- `isLatestPost` - display the latest post.
+- `tags` - display the latest post in specific tags.
```

Comment:
What do you think of combining those two keys? So if you set `tags` it's simply the latest one?